game/app/controllers/scores_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ScoresController < ApplicationController
+    # TODO after javascript training:
+    # Include adding new score to user after playing game.
```

Comment:
@lizgarciao would it be better to create a separate ticket and add the github url here as a TODO?